community/index/src/test/java/org/neo4j/index/internal/gbptree/GBPTreeTest.java

```diff
@@ -28,10 +28,12 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.OpenOption;
```

Comment:
just noticed that index tests create tons of test managed threads, why do we need to do that instead of using any sort of thread pool?
that will also allow you to use Callables and throw exceptions and not use throwing wrapper